addons/test_mail_full/tests/test_mail_performance.py

```diff
@@ -312,7 +313,7 @@ def test_portal_message_format_rating(self):
     def test_portal_message_format_monorecord(self):
         message = self.messages_all[0].with_user(self.env.user)
 
-        with self.assertQueryCount(employee=16):
+        with self.assertQueryCount(employee=19):
```

Comment:
+3 instead of +2 in this test? Why?

Reply:
There's an extra query for partner here, which I think makes sense considering this is a mono record test, so despite others in this case, not all partners were fetched beforehand.